chips/sam4l/src/spi.rs

```diff
@@ -233,7 +232,7 @@ impl spi::SpiMaster for Spi {
         let regs: &mut SpiRegisters = unsafe { mem::transmute(self.registers) };
 
         self.enable_clock();
-
+        self.set_baud_rate(40000);
```

Comment:
exactly, I mean some higher layer should set the baud rate. This could be a process ("application"), it could be a higher-level capsule (like a capsule driver for the rf233) or it could, in this case for now, be the board configuration, but it definitely shouldn't be in the low-level driver.

I'm basically skeptical this file should change at all.